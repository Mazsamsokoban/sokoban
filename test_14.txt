loadtest 14
listworkers
listboxes
move w1 right
showmaze
listworkers
listboxes