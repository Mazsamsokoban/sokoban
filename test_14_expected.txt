f0:w1 f1:box1 f2:w2 f3:box2 bf:null 

name:w1
field:f0
direction:Neutral
points:0
pushforce:10.0

name:w2
field:f2
direction:Neutral
points:0
pushforce:10.0

name:box1
field:f1
direction:Neutral
owner:null
pusher:null

name:box2
field:f3
direction:Neutral
owner:null
pusher:null

f0:null f1:w1 f2:box1 f3:w2 bf:box2 

name:w1
field:f1
direction:Right
points:0
pushforce:10.0

name:w2
field:f3
direction:Right
points:1
pushforce:10.0

name:box1
field:f2
direction:Right
owner:null
pusher:w1

name:box2
field:bf
direction:Right
owner:w2
pusher:w2

